packages/react-scripts/template/src/index.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import * as serviceWorkerRegistration from './service-worker-registration';
```

Comment:
*Just noting that these proposed changes are not in any pr (everything else is though)*